docs/reference/remap/flatten.cue

```diff
@@ -0,0 +1,64 @@
+package metadata
+
+remap: functions: flatten: {
+	arguments: [
+		{
+			name:        "value"
+			description: "The array or map to flatten."
+			required:    true
+			type: ["array", "map"]
+		},
+	]
+	return: ["array", "map"]
+	category: "text"
+	description: #"""
+			Returns a nested array or map that has been flattened to a single level.
+		"""#
+	examples: [
+		{
+			title: "Array Flatten"
+			input: {
+				array: [1, [2, 3, 4], [5, [6, 7], 8], 9]
+			}
+			source: #"""
+				.flattened = flatten(array)
+				"""#
+			output: {
+				flattened: [1, 2, 3, 4, 5, 6, 7, 8, 9]
+			}
+		},
+		{
+			title: "Map Flatten"
+			input: {
+				object: {"grandparent": {"parent1": {"child1": 1
+										"child2": 2
+				}
+					"parent2": {"child1": 3}
+				}
+				}
```

Comment:
some weird indenting going on here. Or is this `cue fmt` doing its thing?

Reply:
This is `cue fmt`.